### ISSUE TITLE
Result after execution CapsNet on same dataset

### ISSUE THREAD

**Issue by birajaghoshal** [thibo73800/capsnet_traffic_sign_classifier#7]
[Train] Batch ID = 0, loss = 4.22318, acc = 0.0
[Validation] Batch ID = 0, loss = 1.2876, acc = 0.02
Evaluate full validation dataset ...
Current loss: 1.4351 Best loss: None
[TOTAL Validation] Batch ID = 0, loss = 1.4351, acc = 0.037641723356

**Comment by thibo73800** [comment]
Hi, what's your question ?
